Resources/Prototypes/_White/Entities/Clothing/Eyes/goggles.yml

```
@@ -61,24 +62,26 @@
-  - type: UserInterface
-    interfaces:
-      enum.ChameleonUiKey.Key:
-        type: ChameleonBoundUserInterface
+  - type: Sprite
+    sprite: _White/Clothing/Eyes/Goggles/sthermals.rsi
+  - type: Clothing
+    sprite: _White/Clothing/Eyes/Goggles/sthermals.rsi
+  - type: ThermalVision
+    flashDurationMultiplier: 2
+    isEquipment: true
+    type: ToggleThermalVisionEvent {}
```

yamllint (1.35.1)

[error] duplication of key "type" in mapping

(key-duplicates)